web/src/user/LibraryPage/ak-library-impl.ts

```diff
@@ -162,10 +170,9 @@ export class LibraryPage extends AKElement {
                 ?isadmin=${this.isAdmin}
             ></ak-library-application-empty-list>`;
         }
-        if (this.filteredApps.length === 0) {
-            return html`<ak-library-application-search-empty></ak-library-application-search-empty>`;
-        }
-        return this.renderApps();
+        return this.filteredApps.some(appHasLaunchUrl) // prettier-ignore
```

Comment:
This is what was missing from the commit.  Dammit.  Also, changed from using `find` in the pre-commit version to `some` here. `Some` is more efficient (doesn't allocate a reference), is type-correct (I want a true/false value, not `app`/undefined) and microseconds faster.  :-)